package.json

```diff
@@ -123,9 +124,11 @@
     "jest-environment-jsdom": "^29.6.4",
     "jest-runner-prettier": "^1.0.0",
     "prettier": "^3.0.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
We definitely do not want this included in here.
We use peerDependcies for React in this project.

```
 "peerDependencies": {
    "react": "^18.0.0",
    "react-dom": "^18.0.0"
  },
```